unitTests/ui/OmniboxTest.ts

```diff
@@ -28,6 +31,38 @@ export function OmniboxTest() {
       Simulate.removeJQuery();
     });
 
+    function setupEnv() {
+      testEnv = new Mock.MockEnvironmentBuilder();
+      testEnv.searchInterface.getOmniboxAnalytics = jasmine.createSpy('omniboxAnalytics').and.returnValue(setupOmniboxAnalytics()) as any;
+    }
+
+    function setupOmniboxAnalytics(): IOmniboxAnalytics {
```

Comment:
I would prefer something along the lines of `buildOmniboxAnalyticsMock` for the name,